src/mono/mono/metadata/class-internals.h

```diff
@@ -587,8 +587,8 @@ typedef struct {
 	// have both of them to be non-NULL.
 	const char *name;
 	gconstpointer func;
-	gconstpointer wrapper;
-	gconstpointer trampoline;
+	gconstpointer wrapper__;
+	gconstpointer trampoline__;
```

Comment:
Ignore the renamed fields. I will remove this.  Just want to see the build fail if there's any #ifdef uses of these that I missed